src/blog/2018-12-24/anouncing-stryker-0-34.pug

```diff
@@ -0,0 +1,128 @@
+        to change. For example:
+        
+        ```shell
+        npx stryker run --testRunner mocha --mutator javascript
+        ```
+
+        Again, special thanks to [Diego R.B.](https://twitter.com/DiegoRBaquero) for inspiring us to implement this.
+
+        With this feature in place, Stryker is now effectively a drop in replacement for [mutode](https://github.com/TheSoftwareDesignLab/mutode).
+
+        ## Init presets
+
+        With `stryker init` (either using the [stryker-cli](https://github.com/stryker-mutator/stryker-cli), or using `npx stryker init`) 
+        you can initialize Stryker in your repository. It asks questions via a questionnaire, like "Which test runner do you want to use?" and
+        "Which test framework do you want to use?". Sometimes you might not know the answer to those questions. It also might take a lot more 
+        configuration to get you started. This is why we added a [guides section to the stryker handbook](https://github.com/stryker-mutator/stryker-handbook/tree/master/stryker/guides).
+        We've now integrated those into the `stryker init` command.
+
+        ```shell
+        $ stryker init
+        ? Are you using one of these frameworks? Then select a preset configuration. (Use arrow keys)
+        > angular-cli
+        react
+        vueJs
+        ──────────────
+        None/other
+        ```
+
+        Choosing for a preset immediately gets you started! It creates the configuration file and installs the correct plugins.
+        Choosing `None/other` here will still take you to the normal questionnaire.
+
+        Special thanks to [wmaarts](https://github.com/Wmaarts) for adding this feature.
+
+        ## Web component tester plugin
+
+        With this release of Stryker, we've also released version 0.1.0 of the stryker-wct-runner. Install it with:
+
+        ```shell
+        $ npm install --save-dev stryker-wct-runner
+        ```
+
+        Configure it with:
+
+        ```js
+        {
+            testRunner: 'wct'
+        }
+        ```
+
+        With this plugin, we now support [Polymer](https://www.polymer-project.org/) cli projects. 
+        The [web-component-tester](https://www.npmjs.com/package/web-component-tester) runs your tests in an actual browser. 
+        However, it doesn't support any of the coverage analysis performance features for now. If you want those to be supported,
+        please open an issue. You're also welcome to take a look at the [issues we've opened at Polymer side](https://github.com/Polymer/tools/issues/created_by/nicojs).
+        Any help there is really appreciated.
+
+        ## What's next?
+
+        We're still focussing on getting v1 out of the door. We're reasonably certain it will happen in Q1 of next year.
+        V1 will mostly be removing deprecated features and an introduction of SemVer with possibly a release cadence. 
```

Comment:
Honestly, I don't get this part. Especially last sentence. I think `and an introduction of SemVer with possibly a release cadence.` Should be rebuilt somehow

Reply:
Agreed. Doesn't Stryker already use SemVer? Unless you mean something like a stricter following of SemVer. Release cadence might be clearer as 'a more predictable release schedule'